flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -474,10 +501,6 @@ private static void validateFlinkTable(CatalogBaseTable table) {
         throw new UnsupportedOperationException("Creating table with computed columns is not supported yet.");
       }
     });
-
-    if (!schema.getWatermarkSpecs().isEmpty()) {
-      throw new UnsupportedOperationException("Creating table with watermark specs is not supported yet.");
```

Comment:
As I mentioned [here](https://github.com/apache/iceberg/issues/2264#issuecomment-877808285) before, maybe we just need to remove this validation, like @kingeasternsun 's [comment](https://github.com/apache/iceberg/pull/3681#issuecomment-1064887971) said.